docs/_articles/en/troubleshooting.md

```diff
@@ -74,3 +74,11 @@ The annotated JSON below shows you what you should remove
 The ESLint engine requires that any typescript files that are scanned must be included by the tsconfig. [More Information](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v2.0.0)
```

Comment:
The red highlighting is a github.com oddity because of the json comments above. You can view the end result here.

https://github.com/forcedotcom/sfdx-scanner/blob/d/jbartolotta/W-7802169/docs/_articles/en/troubleshooting.md